tools/Azure.Mcp.Tools.Foundry/src/Commands/KnowledgeIndexListCommand.cs

```diff
@@ -66,11 +66,9 @@ public override async Task<CommandResponse> ExecuteAsync(CommandContext context,
                 options.Tenant,
                 options.RetryPolicy);
 
-            context.Response.Results = indexes?.Count > 0 ?
-                ResponseResult.Create(
-                    new KnowledgeIndexListCommandResult(indexes),
-                    FoundryJsonContext.Default.KnowledgeIndexListCommandResult) :
-                null;
+            context.Response.Results = ResponseResult.Create(
+                new KnowledgeIndexListCommandResult(indexes ?? []),
```

Comment:
Is this a new design choice for null enumerables to return an empty array instead of null?